app/utils/print/playback/sections/LeadTrusteeBusiness.scala

```diff
@@ -33,13 +32,14 @@ object LeadTrusteeBusiness {
       Some(Seq(AnswerSection(
         headingKey = Some(messages("answerPage.section.leadTrustee.subheading")),
         Seq(
-          stringQuestion(TrusteeOrgNamePage(index), userAnswers, "trusteeBusinessName"),
           yesNoQuestion(TrusteeUtrYesNoPage(index), userAnswers, "leadTrusteeUtrYesNo", name),
```

Comment:
Where is the "Is this trustee a UK registered business?" yes/no question?

Reply:
As per Rachel's feed back:
Lead trustee - The 'Is trustee a UK registered company/' is the first question.
The TrusteeUtrYesNo renders 'Is trustee a UK registered company?'